src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -135,7 +135,8 @@ void AlertManeuverRequest::on_event(const event_engine::Event& event) {
   LOG4CXX_AUTO_TRACE(logger_);
   using namespace helpers;
   const smart_objects::SmartObject& message = event.smart_object();
-
+  bool is_tts_ok, is_no_navi_error;
+  hmi_apis::Common_Result::eType tts_result, navi_result;
```

Comment:
@OHerasym Every variable must be declared on separate line. Even of the same type.